fr.openfisca.org/api/latest/deploy.sh

```diff
@@ -5,7 +5,7 @@ set -ex
 REQUIREMENTS="/home/openfisca/openfisca-ops/fr.openfisca.org/api/latest/requirements.txt"
 
 source /home/openfisca/virtualenvs/api-fr-latest/bin/activate
-pip install --requirement "$REQUIREMENTS"
+pip install --requirement "$REQUIREMENTS" --upgrade --upgrade-strategy eager
```

Comment:
These options actually make pip behave like a clean environment: it will install the latest versions of all packages, without taking into consideration what was previously installed.